src/components/Modal/Modal.tsx

```diff
@@ -22,7 +23,7 @@ export type ModalProps = {
   isContentPadded?: boolean;
 };
 
-const Modal: React.FCC<ModalProps> = ({
+const Modal: React.FC<React.PropsWithChildren<ModalProps>> = ({
```

Comment:
this could probably be FCC

Reply:
fixed here https://github.com/Orfium/orfium-ictinus/pull/720/commits/a4a6b5364393adb1f2351a633b714e8d7d8eebdc